modules/social_features/social_core/src/SocialCoreServiceProvider.php

```diff
@@ -29,6 +30,14 @@ public function alter(ContainerBuilder $container) {
         $definition->setClass('Drupal\social_core\Entity\Select2EntityAutocompleteMatcher');
       }
     }
+
+    // Replaces all EDA Handlers with dummies if there is no Publisher.
+    if (!$container->hasDefinition(SocialEdaDispatcher::class)) {
```

Comment:
I think we should use `@phpstan-ignore` here. The goal of our baseline is to ignore historical errors which we plan to eventually fix. The PHPStan "error" we're introducing on this line is on purpose and not one we expect someone to fix in the future. Annotating it explicitly makes that clear and prevents someone trying to fix it from the baseline in the future.

```suggestion
    // In this case we expect the class not to be found because it exists outside of
    // the Open Social distribution.
    // @phpstan-ignore class.notFound
    if (!$container->hasDefinition(SocialEdaDispatcher::class)) {
```

Reply:
Great, thanks :) I added it to my commit.